avaframe/com1DFA/com1DFA.py

```diff
@@ -2670,6 +2700,8 @@ def getSimTypeList(standardCfg, simTypeList, inputSimFiles):
             simTypeList.append("ent")
         elif entResInfo["flagEnt"] == "No" and entResInfo["flagRes"] == "Yes":
             simTypeList.append("res")
+        elif entResInfo["flagDet"] == "Yes":
+            simTypeList.append("det")
```

Comment:
@awirb is this really needed? Because if we see detrainment as part of our resistance implementation, then we would not need this, right?

Reply:
I don't think that it is needed. It was from the version before, where I added a simType 'detrainment'.